tests/bitcoin/BitcoinClient.spec.ts

```diff
@@ -1037,7 +1037,7 @@ describe('BitcoinClient', async () => {
         labels: [
           {
             name: 'sidetree',
-            purpose: 'receive'
+            verificationRelationship: 'receive'
```

Comment:
Find and replace error.

Reply:
https://github.com/decentralized-identity/sidetree/pull/881/commits/cba9d69241087b4f4b2c4803aaa9b8d3431538ca